### PR TITLE
Added Semantic Versioning and debug statements

### PR DESCRIPTION
**Pull request by mariamcws**
## Description
Implement Semantic Versioning Support for cwds-jobs

## Jira Story
[Implement Semantic Versioning Support for CALS-Job](https://osi-cwds.atlassian.net/browse/FIT-291)

## Types of changes
- [x] New feature (non-breaking change for continuous integration)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -53,7 +53,15 @@ project.ext {
     allTestCoverageFile = "$buildDir/jacoco/allTestCoverage.exec"
 }
 
-version = projectVersion
+//version = projectVersion
```
Comment (edgibbs) [inline build.gradle]: Can probably remove the comment.

File: build.gradle [hunk 2]
```diff
@@ -53,7 +53,15 @@ project.ext {
...
+if (System.getProperty('newVersion')) {
+    def newVersion = "${System.getProperty('newVersion')}"
+    version = newVersion
+    print version + " With newTag"
```
Comment (edgibbs) [inline build.gradle]: ⛔️ REQUIRED: I think you have to use echo for this to work, there's no Jenkins DSL print.  I tried to find if it actually showed up in the console logs, but they're 60MB and almost impossible to parse.

File: build.gradle [hunk 3]
```diff
@@ -53,7 +53,15 @@ project.ext {
...
+} else {
+    version = projectVersion
+    print version + " WithOut newTag"
```
Comment (edgibbs) [inline build.gradle]: 🚫 REQUIRED: Same thing here.

File: build.gradle (hunk 1 above)
Comment (mariamcws) [reply build.gradle]: removed

File: build.gradle (hunk 2 above)
Comment (mariamcws) [reply build.gradle]: fixed

File: build.gradle (hunk 3 above)
Comment (mariamcws) [reply build.gradle]: fixed

**Pull request merged**